On this page highlight fix scroll and href fix

tino-liu opened the pull request:
This PR fixes the following points:
- On this page items not highlighting correctly on screen scroll
- Item click aren't scrolling to its content
- Highlight isn't properly showing based on screen scroll position

isaaclombardssw reviewed (approved):
LGTM – nice

Get test pass for the UX, Betty maybe? LGTM though

tino-liu commented:
As per my conversation with @bettybondoc ,
Added a bit more padding to indented item and moved entire module down to be on the same line with the doc title
